R/list.R

```diff
@@ -6,10 +6,36 @@
 #'   [validate_email()].
 #' @param package `NULL`, or a character scalar. Can be used to restrict
 #'   the search for a single package.
-#' @param howmany How many check groups (checks submitted simultaneously) to show. The current API limit is 20.
-#' @return An [`rhub_check_list`] object.
+#' @param howmany How many check groups (checks submitted simultaneously)
+#'   to show. The current API limit is 20.
+#' @return A tibble with columns:
```

Comment:
should tibble here refer to tibble docs i.e. `[tibble::tibble]`? It'd look good in the pkgdown website at least.